login/src/Components/RegisterBody/RegisterBody.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import img1 from "../../images/software.jpg"
+
+
+export default function RegisterBody() {
+  const validateform=(e)=>{
+    e.preventDefault()
+
+    let email=document.getElementById('InputEmail').value
+    let pattern = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
please create a validation file in the util folder and keep all validations in that file.